internal/pgbackrest/reconcile.go

```diff
@@ -206,6 +206,12 @@ func AddConfigToRestorePod(
 	sources := append([]corev1.VolumeProjection{},
 		cluster.Spec.Backups.PGBackRest.Configuration...)
 
+	if cluster.Spec.DataSource != nil &&
```

Comment:
Like, if you were mounting the same secret as both a Backups configuration and as a DataSource configuration? I've actually experimented with this case (e.g., bootstrapping from one S3 bucket and pushing backups to another, using the same s3 secret), but I didn't actually see any issue with that... Source is a list, so they all get mounted (theoretically), but then they all get mounted to the same location and one overwrites the other....

But is it always the DataSource that overwrites the Backup (which we'd want since this is the RestorePod...

Reply:
Oh, wait, according to the comment above line 206, yeah, last write wins